scripts/index.js

```diff
@@ -154,6 +156,20 @@ fs.emptyDir(paths.appBuild)
           console.log();
         }
 
+        if (postbuild) {
+          exec(postbuild, (error, stdout, stderr) => {
+            if (error) {
+              console.log(`error: ${error.message}`);
+              return;
+            }
+            if (stderr) {
+              console.log(`stderr: ${stderr}`);
+              return;
+            }
+            console.log(`stdout: ${stdout}`);
+          });
+        }
+
```

Comment:
I'd rather chain that as another then after this line: https://github.com/Nargonath/cra-build-watch/blob/acb74b48ffe39b25f68fa2ce73f6f93969946ee8/scripts/index.js#L161. This way your build path will also contain the `public` folder content.

Also as you're using the asynchronous version of `exec`, please wrap in a `Promise` to follow the current async chain.